pkgs/tools/system/openipmi/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchurl, popt, ncurses, python, readline, lib }:
+
+stdenv.mkDerivation rec {
+  pname = "OpenIPMI";
+  version = "2.0.31";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/openipmi/${pname}-${version}.tar.gz";
+    sha256 = "05wpkn74nxqp5p6sa2yaf2ajrh8b0gfkb7y4r86lnigz4rvz6lkh";
+  };
+
+  buildInputs = [ popt python ncurses readline ];
+  outputs = [ "dev" "bin" "out" "man" ];
```

Comment:
Does it make sense to split the library? We have no consumers atm.

Reply:
No _in-nixpkgs_ consumers.

Reply:
I would like to add openipmi as an optional dependency to collectd.